### PR TITLE
Rekeying function update

### PR DESCRIPTION
**Pull request by russau**
Fix for json escaping, and reserved word in the statements.sql

### CODE REVIEW

File: _includes/tutorials/rekeying-function/ksql/code/tutorial-steps/prod/send-to-api.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 tr '\n' ' ' < src/statements.sql | \
 sed 's/;/;\'$'\n''/g' | \
 while read stmt; do
-    echo '{"ksql":"'$stmt'", "streamsProperties": {}}' | \
+    jq --arg value1 "$stmt" -n '{ "ksql" : $value1 }'| \
```
Comment (gAmUssA) [inline _includes/tutorials/rekeying-function/ksql/code/tutorial-steps/prod/send-to-api.sh:4]: I'm not sure about jq.
We need to change rest of the tutorial for people to install all required dependencies

Comment (colinhicks) [reply _includes/tutorials/rekeying-function/ksql/code/tutorial-steps/prod/send-to-api.sh:4]: We were already requiring it. This change adds a second usage in the same script. We can follow up in another issue to cover jq as a prerequisite.

Comment (russau) [reply _includes/tutorials/rekeying-function/ksql/code/tutorial-steps/prod/send-to-api.sh:4]: We use jq in the current template @ `templates/ksql/static/prod/code/send-to-api.sh`.

Comment (gAmUssA) [reply _includes/tutorials/rekeying-function/ksql/code/tutorial-steps/prod/send-to-api.sh:4]: Got it. Thanks

**Pull request merged**